Dockerfile

```diff
@@ -0,0 +1,65 @@
+# This Dockerfile is for development of COSMOS
+# See Docker Hub: ballaerospace/cosmos for production docker images
+
+FROM ubuntu:18.04
+
+RUN apt-get update -y && apt-get install -y \
+  cmake \
+  default-jdk \
+  freeglut3 \
+  freeglut3-dev \
+  gcc \
+  g++ \
+  git \
+  iproute2 \
+  libffi-dev \
+  libgdbm-dev \
+  libgdbm5 \
+  libgstreamer-plugins-base1.0-dev \
+  libgstreamer1.0-dev \
+  libncurses5-dev \
+  libreadline6-dev \
+  libsmokeqt4-dev \
+  libssl-dev \
+  libyaml-dev \
+  net-tools \
+  postgresql-server-dev-all \
+  qt4-default \
+  qt4-dev-tools \
+  ruby2.5 \
+  ruby2.5-dev \
+  vim \
+  xterm \
+  zlib1g-dev
+
+RUN gem install rake --no-document
```

Comment:
Each RUN, COPY, ADD instruction adds a Docker layer so it's best to combine these where possible with &&. Obviously that can hurt readability as well so it's a trade off. I'd move this line farther down.